### PR TITLE
test: reset password and then continue on universal login 

### PR DESCRIPTION
**Pull request by danielgent**
A precursor to whatever change in functionality we decide to stop the user reaching a dead end

### CODE REVIEW

File: test/integration/login/forgot-password.spec.ts [hunk 1]
```diff
@@ -106,10 +110,48 @@ describe("Forgot password", () => {
...
+
+    expect(resetPasswordSubmission.status).toBe(200);
+
+    await snapshotResponse(resetPasswordSubmission);
+
+    // ---------------------
+    // login with the new password using universal login
+    // ---------------------
+
+    // TODO - should the user be redirected straight to this page to login and continue the flow?
```
Comment (danielgent) [inline test/integration/login/forgot-password.spec.ts:138]: :eyes: On the previous step we show a message saying _you have successfully reset your password_ should we  add a button that then sends them straight to the enter password form (continuing their flow)?

**Review by markusahlstrand — APPROVED** [review]
Looks good. I think we maybe should remove the "Framgång" text. It's not easy to translate it to something that makes sense and think the text below probably is sufficient?

**Comment by danielgent** [comment]
> Looks good. I think we maybe should remove the "Framgång" text. It's not easy to translate it to something that makes sense and think the text below probably is sufficient?

Let me know whatever you want changing anywhere. I would think we want more text, but then people don't read text anyway :smile:

**Comment by danielgent** [comment]
> Looks good. I think we maybe should remove the "Framgång" text. It's not easy to translate it to something that makes sense and think the text below probably is sufficient?

Happy to go with whatever anyone thinks for any of these pages :+1:

**Pull request merged**